src/parser/mod.rs

```diff
@@ -9342,12 +9342,37 @@ impl<'a> Parser<'a> {
         optional: IsOptional,
         allow_empty: bool,
     ) -> Result<Vec<Ident>, ParserError> {
+        self.parse_parenthesized_column_list_inner(optional, allow_empty, |p| p.parse_identifier())
+    }
+
+    /// Parses a parenthesized comma-separated list of unqualified, possibly quoted identifiers
+    pub fn parse_parenthesized_qualified_column_list(
+        &mut self,
+        optional: IsOptional,
+        allow_empty: bool,
+    ) -> Result<Vec<ObjectName>, ParserError> {
+        self.parse_parenthesized_column_list_inner(optional, allow_empty, |p| {
+            p.parse_object_name(false)
```

Comment:
Should the `in_table_clause` be configurable here? I thinking it's supposed to be set to true when parsing a `USING` clause given the behavior being fixed